src/external/monaco.flux.messages.ts

```diff
@@ -167,13 +167,16 @@ export const parseResponse = (response: ServerResponse): LSPResponse => {
     throw new Error(error)
   }
 
-  const split = (message || '').split('\r\n')
+  if (message) {
```

Comment:
new lsp library will return an undefined `message` so we need to handle that here

Reply:
For other reviewers: in the spec, some requests don't have responses. The old LSP would just spit some empty JSON over the wire, but the new LSP doesn't respond with anything if nothing is expected.